libraries/ms-common/rules.mk

```diff
@@ -8,5 +8,5 @@
 $(T)_DEPS := $(PLATFORM_LIB) libcore
 
 ifeq (x86,$(PLATFORM))
-$(T)_EXCLUDE_TESTS := can can_hw adc
+$(T)_EXCLUDE_TESTS := can can_hw adc pwm
```

Comment:
I was thinking more something along the lines of a socket or pipe that we can use to set states and variables. I'd like to be able to test application code as much as possible, and right now it's looking pretty difficult to write tests for different conditions.